howso/utilities/feature_attributes/time_series.py

```diff
@@ -247,6 +303,27 @@ def _set_rate_delta_bounds(self, btype: str, bounds: Dict, features: Dict):
                     else:
                         warnings.warn(f"Ignoring {btype}_boundaries for order {order}: out of range")
 
+    @staticmethod
```

Comment:
This is doing the same thing as `np.array_split(df, n)`, let's use that instead of defining a new function.